### PR TITLE
backend: add health to telemetry

### LINKED ISSUE #333: Implement telemetry health in backend

### PR DESCRIPTION
**Pull request by JonasVautherin**
Fixes #333.

### CODE REVIEW

File: backend/test/telemetry_service_impl_test.cpp [hunk 1]
```diff
@@ -141,28 +178,128 @@ Position TelemetryServiceImplTest::createPosition(const double lat, const double
...
+
+    checkSendsPositions(positions);
+}
+
+TEST_F(TelemetryServiceImplTest, registersToTelemetryHealthAsync)
+{
+    EXPECT_CALL(*_telemetry, health_async(_))
+    .Times(1);
+
+    std::vector<Health> healths;
```
[the remark this answers is not in the document]
Comment (JonasVautherin) [reply backend/test/telemetry_service_impl_test.cpp]: I'm not sure I understand the question. I collect the `Health` events sent by the drone in this vector, for the test...

How would you do it?

File: plugins/telemetry/telemetry.cpp [hunk 2]
```diff
@@ -317,4 +317,25 @@ std::ostream &operator<<(std::ostream &str, Telemetry::Position const &position)
...
+           && lhs.local_position_ok == rhs.local_position_ok
+           && lhs.global_position_ok == rhs.global_position_ok
+           && lhs.home_position_ok == rhs.home_position_ok;
+}
+
+std::ostream &operator<<(std::ostream &str, Telemetry::Health const &health)
+{
+    return str << "[gyrometer_calibration_ok: " << health.gyrometer_calibration_ok <<
+           ", accelerometer_calibration_ok: " << health.accelerometer_calibration_ok <<
+           ", magnetometer_calibration_ok: " << health.magnetometer_calibration_ok <<
```
Comment (julianoes) [inline plugins/telemetry/telemetry.cpp]: I usually start these lines with `<<` and indent it in the same way. Just for next time.

Comment (JonasVautherin) [reply plugins/telemetry/telemetry.cpp]: I did not think about it and let astyle do it. But I prefer you're way. I'll change it.

Comment (JonasVautherin) [reply plugins/telemetry/telemetry.cpp]: Well, I'm actually moving that in the next PR, so I'll change it at this time :-).

**Pull request merged**